tests/test-sources/plugins/null-ls.nix

```diff
@@ -36,31 +34,37 @@
       updateInInsert = false;
       sources = {
         code_actions = {
+          eslint.enable = true;
+          eslint_d.enable = true;
           gitsigns.enable = true;
           shellcheck.enable = true;
           statix.enable = true;
         };
         diagnostics = {
           cppcheck.enable = true;
+          deadnix.enable = true;
+          eslint.enable = true;
+          eslint_d.enable = true;
           flake8.enable = true;
           gitlint.enable = true;
           shellcheck.enable = true;
           statix.enable = true;
-          deadnix.enable = true;
         };
         formatting = {
```

Comment:
It seems that you forgot to enable `prettier_d` in the tests

Reply:
Good catch! Thanks